src/main/views/macro/financialDetails.njk

```diff
@@ -132,8 +132,20 @@
   {% endif %}
 {% endmacro %}
 
+{% macro priorityDebtsSection(debts) %}
+   {{ row(label = 'Do they have any debts they are behind on?', value = (debts !== undefined) | renderYesNo, bottomBorder = not debts) }}
+  {% for debt in debts %}
+    {% if debts.length === 1 %}
+      {{ row(label = 'Debt' , value = debt.type | renderPriorityDebtType | capitalize) }}
+    {% else %}
+      {{ row(label = loop.index0 + 1 + '. Debt' , value = debt.type | renderPriorityDebtType | capitalize) }}
```

Comment:
`loop.index0 + 1` is the same as `loop.index`

Reply:
Although I agree with you, this seems to be the syntax being used through out the code.

Reply:
Now's your chance to introduce a better way of coding then! ;)